code/datums/components/fantasy/suffixes.dm

```diff
@@ -189,22 +195,79 @@
 	name = "curse of hunger"
 	placement = AFFIX_SUFFIX
 	alignment = AFFIX_EVIL
+	weight = 5
 
 /datum/fantasy_affix/curse_of_hunger/validate(obj/item/attached)
-	//curse of hunger that attaches onto food has the ability to eat itself. it's hilarious.
-	if(!IS_EDIBLE(attached))
```

Comment:
Is this no longer an issue?

Reply:
it was returning TRUE anyways so the if statement was redundant